confidence: 0
seconds_per_order: 4
appid: 60166bfe
